packages/pcd/gpc-pcd/src/GPCPCDPackage.ts

```diff
@@ -274,39 +272,49 @@ export function getProveDisplayOptions(): ProveDisplayOptions<GPCPCDArgs> {
       pods: {
         argumentType: ArgumentTypeName.Record,
         description: "Generate a proof for the selected POD object",
-        validate:
-          (podName) =>
-          (podPCD, params): boolean => {
-            if (podPCD.type !== PODPCDTypeName) {
+        validate: (podName, podPCD, params): boolean => {
+          if (podPCD.type !== PODPCDTypeName) {
+            return false;
+          }
+
+          if (params?.proofConfig !== undefined) {
+            const proofConfig = ((): GPCProofConfig | undefined => {
+              try {
+                return deserializeGPCProofConfig(params.proofConfig);
+              } catch (e) {
+                if (e instanceof TypeError) {
+                  params.notFoundMessage = e.message;
+                }
+              }
```

Comment:
If this isn't a TypeError, it's falling through to default-return undefined.  That's pretty non-obvious.  I'd recommend an explict return, or `throw e;` instead.